raiden/src/ops/transact_write.rs

```diff
@@ -89,6 +89,12 @@ impl WriteTx {
             .await
     }
 
+    #[cfg_attr(feature = "tracing", tracing::instrument(
+        level = tracing::Level::DEBUG,
+        name = "dynamodb::action",
+        skip_all,
+        fields(api = "transact_write_items")
+    ))]
```

Comment:
Make span at WriteTx::inner_run to wrap calling TransactWriteItems API.  
This span does not support field for table name.